packages/client/src/checkout/getOperation.ts

```diff
@@ -0,0 +1,40 @@
+import client, { adaptError } from '../helpers/client';
+import join from 'proper-url-join';
+import type { GetOperation } from './types/getOperation.types';
+
+/**
+ * @typedef {object} GetOperationParams
+ *
+ * @alias GetOperationParams
+ * @memberof module:checkout
+ *
+ * @property {number} orderId - Identify the checkout order.
+ * @property {string} operationId - Identify the operation.
+ */
+
+/**
+ * Method responsible for fetching order operation.
+ *
+ * @function getOperation
+ * @memberof module:checkout
+ *
+ * @param {GetOperationParams} [params] - Fetch params.
```

Comment:
```suggestion
 * @param {GetOperationParams} params - Fetch params.
```

I think `params` cannot be optional 🤔

Reply:
✅